auth/identity_context.go

```diff
@@ -81,6 +86,16 @@ func (c IdentityContext) AuthenticatedAt() time.Time {
 	return c.authenticatedAt
 }
 
+func (c IdentityContext) UserIdentifier() string {
+	return c.userIdentifier
+}
+
+// SetUserIdentifier allows you to explicitly set user identifier
+func (c *IdentityContext) SetUserIdentifier(id string) IdentityContext {
```

Comment:
does this need to return something?  can it just not return anything?

Reply:
changed